src/resources/refund.js

```diff
@@ -0,0 +1,44 @@
+import T from 'proptypes';
+import base from './base';
+
+export const propTypes = {
+  id: T.string,
+  tracking_code: T.string,
+  confirmation_number: T.string,
+  status: T.string,
+  carrier: T.string,
+  shipment_id: T.string,
+};
+
+export default (api) =>
+  class Refund extends base(api) {
+    static _name = 'Refund';
+
+    static _url = 'refunds';
+
+    static key = 'refund';
+
+    static propTypes = propTypes;
+
+    /**
+     * create a refund
+     * @returns {Promise<never>}
+     */
+    static async save(params = {}) {
+      const newParams = { refund: params };
+      try {
+        const response = await api.post(this._url, { body: newParams });
+        return response.body;
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+
+    /**
+     * delete not implemented
+     * @returns {Promise<never>}
+     */
+    static delete() {
+      return this.notImplemented('delete');
```

Comment:
Why is this not implemented?

Reply:
The `refund` class should not have `delete()` method. The parent class `base` has the `delete()` method, so we need to override it.

Reply:
Yeah, this is a standard pattern in this lib.